src/containers/donation/index.css

```diff
@@ -1,75 +1,76 @@
-.secondaryTitle,
-.introParagraph,
-.donateParagraph,
-.helpParagraph,
-.countNumbers,
-.darkGreyColumn,
-.lightGreyColumn {
+.title h1 {
```

Comment:
be careful with the classname "title" because its very likely one of your teammates is also using it